www/src/Contexts/AppContext.jsx

```diff
@@ -20,7 +20,7 @@ yup.addMethod(
 	'validateSelectionWhenValue',
 	function (name, choices) {
 		return this.when(name, {
-			is: (value) => !isNaN(parseInt(value)),
+			is: (value) => value > 0,
```

Comment:
Instead of value > 0 which will throw a different error for non-numbers, let's make this two checks instead:

something like,

is: (value) => (!isNan(value) ? parseInt(value) : -1)

Reply:
Currently "name" is all about the enabling state of addon, and they should be bound by the yup rule to be of type number. But for future code compatibility, I will modify and test it as you suggest.